.eslintrc.json

```diff
@@ -4,22 +4,27 @@
     "es6": true,
     "jest": true
   },
+  "parser": "babel-eslint",
   "parserOptions": {
-    "ecmaVersion": 2018,
-    "sourceType": "module",
     "ecmaFeatures": {
-      "jsx": true,
-      "modules": true
-    }
+      "jsx": true
+    },
+    "ecmaVersion": 2018,
+    "sourceType": "module"
   },
-  "extends": ["airbnb-base"],
+  "extends": ["airbnb", "plugin:react/recommended"],
+  "plugins": ["react"],
   "rules": {
-    "no-shadow": "off",
-    "no-param-reassign": "off",
-    "eol-last": "off",
-    "arrow-parens": "off",
-	"no-unused-vars": "off",
-	"react/prefer-stateless-function":  [0, { "ignorePureComponents": true }]
+    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+    "import/no-unresolved": "off",
+	"no-shadow": "off",
+	"react/destructuring-assignment": [0],
+	"arrow-parens": ["error", "as-needed"],
+	"react/prop-types":[0],
+	"react/require-default-props":[0],
+	"react/default-props-match-prop-types":[0],
+	"react/no-access-state-in-setstate":[0],
+	"react/button-has-type":[0]
   },
   "ignorePatterns": ["dist/", "build/"]
-}
+}
```

Comment:
- [ ] Your eslintrc file is different from the one provided by Microverse. We should not change this file. We need to switch this file back to its original configuration, you can find it [here](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json).

Check here how you can make your code in accordance if these rules:

- [react/prop-types](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md)

- [react/require-default-props](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md)

- [react/default-props-match-prop-types ](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/default-props-match-prop-types.md)

- [react/no-access-state-in-setstate](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-access-state-in-setstate.md)

- [react/button-has-type](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/button-has-type.md)

You can always try to google any rule that may cause trouble to you. I most cases is really simple to make your code comply with the rule.

Reply:
Have resolved everything, need to update entire logic, Kindly check the working ,eslint & do let me know if everything looks fine on you end, sending for a re-review 💯